dashboard/src/features/shared/components/BaseList/BaseList.jsx

```diff
@@ -10,6 +10,7 @@ class ItemList extends React.Component {
   render() {
     const label = this.props.label || pluralize(humanize(this.props.type))
     const objectName = label.slice(0,-1)
+    const showWelcome = this.props.showWelcome || false
```

Comment:
If we put the `h2` for `showWelcome` in the empty content for the transaction list, we can use one less prop and bit less code.